TWLight/resources/templates/resources/partner_detail.html

```diff
@@ -22,23 +22,24 @@
 {% endblock extra_css %}
 
 {% block content %}
+  <a href="{% url 'partner:filter' %}" class="btn btn-primary">Back to partners</a>
```

Comment:
This could be a `btn-default` rather than `btn-primary` and could be moved beneath `<h1>{{ object }}</h1>` so as to blend even more nicely.

Reply:
Okay!